File: models/Factura.js
export default class Factura {
  constructor(id,identificacion, nombre,numeroFactura,fecha, proveedor, productoId,subproductoId,cantidad,subtotal, iva, total) {
    this.id = id;
    this.identificacion = identificacion;
    this.nombre = nombre;
    this.numeroFactura = numeroFactura;
    this.fecha = fecha;
    this.proveedor = proveedor;
    this.productoId = productoId;
    this.subproductoId = subproductoId;
    this.cantidad = cantidad; 
    this.subtotal = subtotal; 
    this.iva = iva; 
    this.total = total; 
    this.sincronizada = false;
  }
}

File: FacturaApi/Models/Factura.cs
namespace FacturaApi.Models
{
    public class Factura
    {
        public int Id { get; set; }
        public required string Identificacion { get; set; }
        public required string Nombre { get; set; }
        public required string NumeroFactura { get; set; }
        public DateTime Fecha { get; set; }
        public required string Proveedor { get; set; }
        public int ProductoId { get; set; }
        public int SubproductoId { get; set; }
        public int Cantidad { get; set; }
        public decimal Subtotal { get; set; }
        public decimal Iva { get; set; }
        public decimal Total { get; set; }
        public bool Sincronizada { get; set; }
    }
}
